pool/object.go

```diff
@@ -28,6 +28,8 @@ func (p *Pool) actualSigner(signer neofscrypto.Signer) neofscrypto.Signer {
 
 // ObjectPutInit initiates writing an object through a remote server using NeoFS API protocol.
 //
+// Auto-session is enabled by default.
```

Comment:
Maybe "operation is executed within a session automatically created by [Pool] unless parameters explicitly override session settings".

Reply:
Updated